### ISSUE TITLE
Graphic message IN waterfall

### ISSUE THREAD

**Issue by W1SPS** [ExpertSDR3/ExpertSDR3-SUGGESTIONS#81]
I've seen folks send their call sign and other small graphic images that are displayed in the waterfall. I think it was a flex that did it. I think that would be a cool ability for the Sunsdr. Like this: https://g8haj.uk/?page_id=290

**Comment by N8SDR007** [comment]
OLD FLEX POWERSDR allowed this, not the smart SDR

There are a couple 3rd party programs which allow it to be done.

**Comment by W1SPS** [comment]
Yeah, I saw that. I just thought it would be neat native to software. Just to cut down on background software.

**Comment by John-KC2QMA** [comment]
That would be very elegant to have "Waterfall TXT" built in to eSDR3!
Also would be a first in the industry.

**Comment by N8SDR007** [comment]
Its a nice feature

In an older PSDR version this existed, 
It also supports an area of Identification, add a timer that can be configured as well 

and a place to add your call sign- set it to auto mode and technically I think your legal
